src/app/components/home.component.ts

```diff
@@ -300,6 +300,8 @@ export class HomeComponent implements OnInit, AfterViewInit {
 
   tagTypeAhead: string = '';
 
+  dangerousDelete: boolean = false;
```

Comment:
Maybe this was WIP stuff -- this variable isn't needed as you can simply use the _settingsButtons_ `dangerouslyDelete.toggled` boolean state when sending the message through `ipcRenderer` (around line 1910).

I'll remove after merging 👌